src/lib/VirtualScroller.js

```diff
@@ -425,6 +425,21 @@ class VirtualScroller {
 
         return scrollTop <= offsetTop && offsetTop <= scrollTop + this.containerHeight;
     }
+
+    /**
+     * Gets the currently visible list items
+     * @return {Array<HTMLElement>} - the list of current visible list items
+     */
+    getVisibleItems() {
+        if (!this.listEl) {
+            return [];
+        }
+
+        return Array.prototype.slice
```

Comment:
iirc you do this a few types, might be worth abstracting the conversion of the listEl into an array?